reactioncommerce/reaction-admin
# refactor: de-meteorize discount codes view

Resolves #229 
Impact: **minor**
Type: **refactor**

NOTE: Test with this API plugin PR: https://github.com/reactioncommerce/plugin-discount-codes/pull/2

To test with a local plugin(not yet published to NPM):
 
1. create a new `plugins` folder adjacent to the `reaction` folder and clone the plugin to test into it.

2. Remove the plugin from the `package.json` in Reaction, in this case the `plugin-discount-codes`

3. In Reaction `docker-compose.dev.yaml` add the following line under the `volumes` section:

```
      - ../plugins/plugin-discount-codes:/usr/local/src/app/node_modules/@reactioncommerce/plugin-discount-codes
```

4. Stop and start the Reaction API

## Issue
The discounts view still depends on Meteor

## Solution
Remove all Meteor dependencies

## Breaking changes
None

## Testing
1. Add/Update/Delete a discount code
2. Filter discount codes by entering a search keyword
3. Verify above functionality works correctly
